parsec/backend/postgresql/organization.py

```diff
@@ -183,6 +184,36 @@ async def stats(self, id: OrganizationID) -> OrganizationStats:
             workspaces=result["workspaces"],
         )
 
+    async def update(self, id: OrganizationID, **fields: Dict[str, Any]) -> None:
+        """
+        Raises:
+            OrganizationNotFoundError
+            OrganizationError
+        """
+        set = ", ".join([f"{k} = ${k}" for k in fields.keys()])
+        q = Q(
```

Comment:
I didin't think about the SQL injection issue because the schema layer sanitize the request's data. 
If we don't create the `Q` on  the fly, we can't run partial update.
Is it a problem ?

For example, in our case we will never update the expiration_date and the outsider_enabled fields in the same time but we will have to set the both values to update one of them.